sql/src/test/java/io/crate/integrationtests/SnapshotRestoreIntegrationTest.java

```diff
@@ -267,6 +273,55 @@ public void testSnapshotWithMetadataDoesNotDeleteExistingStuff() throws Exceptio
             "foo| 1.2\n"));
     }
 
+    @Test
+    public void testSnapshotWithMetadataModified() throws Exception {
```

Comment:
tbh, I am not really happy with this testcase, neither the way it looks nor the way it tests.

Reply:
Looks okay to me